models/action.go

```diff
@@ -245,12 +245,12 @@ type PushCommits struct {
 	Commits    []*PushCommit
 	CompareUrl string
 
-	avatars map[string]string
+	Avatars map[string]string
 }
 
 func NewPushCommits() *PushCommits {
 	return &PushCommits{
-		avatars: make(map[string]string),
+		Avatars: make(map[string]string),
```

Comment:
This was uppercased to make it Public so other routers could make a PushCommits, such as line 66 of routers/repo/branch.go